### PR TITLE
Add cross-section models

### PR DESCRIPTION
**Pull request by dbrakenhoff**
- add LineSink1DBase, DischargeLineSink1D, LineSink1D and HeadLineSink1D
- add example notebook with comparison to analytical solutions. Both single and multiple aquifer examples shown. 
- add continuity of head and flow elements/equations
- add example notebook for 1d inhom elements

EDIT: update 11/11/2024

### CODE REVIEW

**Comment by mbakker7** [comment]
Very nice start! There are some errors because timml cannot be imported. Maybe store the timml solution separately or should we add a dependency?

File: docs/01howto/index.rst
Comment (dbrakenhoff) [inline docs/01howto/index.rst:1]: fix nb name

File: ttim/linedoublet1d.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        self.dischargeinflayers = np.sum(
+            self.dischargeinf * self.aq.eigvec[self.layers, :, :], 1
+        )
+        self.resfac = self.aq.Haq[self.layers] / self.res
+
+    def setflowcoef(self):
+        """Separate function so that this can be overloaded for other types."""
+        self.flowcoef = 1.0 / self.model.p  # Step function
+
+    def potinf(self, x, _, aq=None):
```
Comment (dbrakenhoff) [inline ttim/linedoublet1d.py:100]: add default value for y?

File: ttim/stripareasink.py [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+                ls="dotted",
+            )
+        else:
+            x2 = ax.get_xlim()[1]
+
+        # water level
+        c = kwargs.pop("color", "b")
+        lw = kwargs.pop("lw", 1.0)
+        ax.plot([x1, x2], [ztop + dy, ztop + dy], lw=lw, color=c, **kwargs)
+        # plot water level symbol: difficult to get consistent, comment out for now
```
Comment (dbrakenhoff) [inline ttim/stripareasink.py:199]: remove this

**Pull request merged**